src/marqo/tensor_search/models/delete_docs_objects.py

```diff
@@ -0,0 +1,24 @@
+"""
+This module holds the classes which define the interface the delete documents
+endpoint.
+"""
+
+import datetime
+from typing import NamedTuple, Literal, List
+
+
+class MqDeleteDocsResponse(NamedTuple):
+    """An object that holds the data we send back to users"""
+    index_name: str
+    status_string: Literal["succeeded"]
```

Comment:
This is because unsuccessful responses result in python exceptions, correct?

Reply:
So far. This is more of a placeholder that has been become a part of the API. There may be a future state where there could be other statuses